docs/changehistory/NextVersion.md

```diff
@@ -62,3 +62,10 @@ There is now a method called `lookAtGlobalLocationFromGcs` on [ViewState3d]($fro
 
 * `rootToCartographicFromGcs` behaves like `rootToCartographic` except it is async and uses the GCS to reproject the location.
 * `cartographicToRootFromGcs` behaves like `cartographicToRoot` except it is async and uses the GCS to reproject the location.
+
+## Formatted property values in ECExpressions
```

Comment:
Let's make this a sub-topic of `## Presentation` to make it easier to understand the context where this function is availab.e

Reply:
Creted sub-topic